packages/devilspie/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://wiki.gnome.org/Projects/DevilsPie
+TERMUX_PKG_DESCRIPTION="Utility for matching X11 windows and performing actions on them"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Rafael Kitover <rkitover@gmail.com>"
+TERMUX_PKG_VERSION=0.23
+TERMUX_PKG_SRCURL=http://deb.debian.org/debian/pool/main/d/devilspie/devilspie_${TERMUX_PKG_VERSION}.orig.tar.xz
+TERMUX_PKG_SHA256=2dd9d4b7424ac999f20c214b0ff71e8c5836d97003d7a8a7391105cd16d36deb
+TERMUX_PKG_DEPENDS="libx11, gettext, glib, gtk3, libwnck"
+
+# These patches are from the Debian package.
+termux_step_patch_package() {
```

Comment:
This function cannot be overridden. Please remove it and let build-package.sh apply patches automatically.

If patches need to be applied in specific order, prepend a number (01, 02, 03, etc) to patch file name.